views/index.jade

```diff
@@ -14,7 +15,7 @@ block content
           input#email.form-control(type='email', placeholder='Enter Email', name='email', required='')
         .form-group
           label.control-label(for='git_url') GITURL:
-          input#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
+          select#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
```

Comment:
How is this select even working. @sch00lb0y  can you please link to a demo deployment